src/Session.cpp

```diff
@@ -65,6 +65,7 @@ const std::vector<Watchable*>& Session::getContent() const
     return content;
 }
 
+template<typename T>
```

Comment:
should it be included both in header file and in here?

Reply:
Yes